### PR TITLE
Add mock community data to dataFiles.html

### LINKED ISSUE #2105: link Bokulich et al mock community test files from the QIIME Resources page
I have all of the links in email. This is a great resource for testing new methods.

### PR DESCRIPTION
**Pull request by colinbrislawn**
Will close #2105 
Still writing, do not merge!

Started by adding qiita links to Bokulich et al, 2013.
Three studies are still missing links to qiita. 
ONLY links to qiita have been added.

### CODE REVIEW

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1698/

**Comment by colinbrislawn** [comment]
Next up, add FTP links from this page.
https://github.com/gregcaporaso/short-read-tax-assignment/blob/master/data/raw-data-urls.txt

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1700/

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1702/

File: home_static/dataFiles.html [hunk 1]
```diff
@@ -105,6 +105,29 @@
...
+			<li>Data set 2, <a href"https://qiita.ucsd.edu/study/description/1685">Qiita study 1685 (private)</a>, <a href="ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/Broad1/">FTP</a></li>
+			<li>Data set 3, <a href="https://qiita.ucsd.edu/study/description/1686">Qiita study 1686</a>, <a href="ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/Broad2/">FTP</a></li>
+			<li>Data set 4, Originally study 1626, now Qiita study <a href="https://qiita.ucsd.edu/study/description/1517">1517 (private)</a></a></li>
+			<li>Data set 5, <a href="https://qiita.ucsd.edu/study/description/1687">Qiita study 1687</a>, <a href="ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/S16S-1/">FTP</a></li>
+			<li>Data set 6, <a href="https://qiita.ucsd.edu/study/description/1688">Qiita study 1688</a>, <a href="ftp://ftp.microbio.me/pub/illumina-mock-communities-raw-data/S16S-2/">FTP</a></li>
+		</ul>
+		<p>The following MiSeq reads were used in Bokulich et al, currentlly <a href="https://peerj.com/preprints/934v2">avaiable as a preprint</a>. Samples are discribed in <a href="https://doi.org/10.7287/peerj.preprints.934v2/supp-1">supp-1</a>.</p>
```
Comment (gregcaporaso) [inline home_static/dataFiles.html]: typo: currentlly

**Comment by gregcaporaso** [comment]
This looks good. I found a couple of typos, could you do a close read of this to see if you spot any more?

**Comment by colinbrislawn** [comment]
Thanks for that Greg. I found another type too. 
I'll squash these commits and then you can merge them. 

Thanks!
Colin

**Comment by colinbrislawn** [comment]
:shipit:

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1704/

**Comment by ekopylova** [comment]
Is it possible to add the expected taxonomies for the mock communities? Ex. for the 1688 dataset,
```
#Taxon MockHiSeq.even MockMiSeq.even Mock.even.phasedmiseq
k__Bacteria;p__Actinobacteria;c__Actinobacteria;o__Bifidobacteriales;f__Bifidobacteriaceae;g__Bifidobacterium 0.041666666 0.041666666 0.041666666
k__Bacteria;p__Actinobacteria;c__Coriobacteriia;o__Coriobacteriales;f__Coriobacteriaceae;g__Collinsella 0.020833333 0.020833333 0.020833333
k__Bacteria;p__Bacteroidetes;c__Bacteroidia;o__Bacteroidales;f__Bacteroidaceae;g__Bacteroides 0.24999999900000003 0.24999999900000003 0.24999999900000003
```

**Comment by gregcaporaso** [comment]
Merged, thanks again @colinbrislawn! 

Looks like there is a minor formatting issue - I'll fix this in another PR.

**Pull request merged**